addons/website_event/tests/test_website_event.py

```diff
@@ -285,3 +285,24 @@ def test_check_search_in_address(self):
 
         with self.assertRaises(AccessError):
             self.env['res.partner'].browse(self.partner.id).read()
+
+    def test_create_website_menu_without_parent_id(self):
+        """Test creating a new website menu item without a parent_id."""
+
+        current_website = self.env['website'].get_current_website()
+        top_menu = current_website.menu_id
+        top_menu.unlink()
+        total_menu_items = self.env['website.menu'].search_count([])
+
+        data = [
+            {
+                'id': "new-1",
+                'parent_id': top_menu.exists().id,  # Deleted the top menu, so parent_id is False
```

Comment:
Here we could maybe directly set it to `False` if we wanted to, not that it will change much more than saving a few lines tho.

Reply:
Hm, yeah it's kind of useless as it stands. I would also avoid the "count check"
imo a test isn't really necessary for this, it never really happens